students/15th/team1/suntae_kim/user_login/views.py

```diff
@@ -0,0 +1,41 @@
+import json
+import os
+import sys
+import bcrypt
+import jwt
+
+from django.shortcuts import render
+from django.http import JsonResponse
+from django.views import View
+
+# sys.path.append(
+#    sys.path.append(os.path.dirname(os.path.abspath(os.path.dirname(__file__))))
+# )
+
+#from suntae_kim.user.models import User
+from user.models import User
+
+class UserLogin(View):
+    def __init__(self):
+        pass
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+            username        = data['username']
+            password        = data['password'].encode('utf-8')
+            user_info       = User.objects.get(username = username)
```

Comment:
info라는 말이 붙지 않아도 user instance에 대한 정보가 담겨있다는 사실을 코드만 보고도 이해할 수 있습니다.
```suggestion
            user       = User.objects.get(username = username)
```